### DISCUSSION TITLE
Querying Payment Routes

### DISCUSSION THREAD

**Discussion by Redinpais** [Ride-The-Lightning/RTL#1111] [Troubleshooting Questions] [answered]
Greetings.
My Umbrel has RTL version outdated (0.12.1-beta) and it supposes show me how to do Querying Payment Routes.
In "Help" section it show how, but this function is not operational.
---
Querying Payment Routes.
Go to the "Transactions" page under the "Lightning" menu :
Query Routes tab is for querying a potential path to a node and a routing fee estimate for a payment amount.
   1. Destination Pubkey - Pubkey of the node, you want to send the payment to.
   2. Amount - Amount in Sats, which you want to send to the node.
----
Thanks

**Comment by saubyk** [comment]
Hi @Redinpais looks like the help section is outdated (created an issue to fix that). Can you try looking under the `Graph Lookups` menu?

**Comment by Redinpais** [comment, marked as answer]
Yes, it's there. Thanks.
